src/tellor_disputables/data.py

```diff
@@ -42,6 +45,19 @@ def get_contract_info(chain_id: int, name: str) -> Tuple[Optional[str], Optional
         return None, None
 
 
+def get_query_type(q: OracleQuery) -> str:
+    """Get query type from class name"""
+    return type(q).__name__
+
+
+def get_contract(web3: Web3, addr: str, abi: str) -> Contract:
+    """Get a contract instance for the given address and ABI."""
+    return web3.eth.contract(  # type: ignore
```

Comment:
This might cause problems bc it's not using our contract class that wraps the eth one, so it won't have the error handling we've written for common rpc errors

Reply:
true

Reply:
Turns out it's not even in use anymore :)